src/mongocxx/write_concern.hpp

```diff
@@ -52,8 +52,6 @@ class uri;
 ///
 /// @see http://docs.mongodb.org/manual/core/write-concern/
 ///
-/// @todo this interface is terrible -- it's exactly what you get with the c-driver essentially.
```

Comment:
Do you think the interface is terrible? I don't know why it's an issue to have it be so similar to the C driver.